examples/server/index.html

```diff
@@ -30,15 +30,15 @@
 
 <h2>Options</h2>
 <div class="option">
-    <input id="use-datachannel" checked="checked" type="checkbox"/>
+    <input id="use-datachannel" type="checkbox"/>
     <label for="use-datachannel">Use datachannel</label>
 </div>
 <div class="option">
-    <input id="use-audio" checked="checked" type="checkbox"/>
+    <input id="use-audio" type="checkbox"/>
     <label for="use-audio">Use audio</label>
 </div>
 <div class="option">
-    <input id="use-video" type="checkbox"/>
+    <input id="use-video" checked="checked" type="checkbox"/>
```

Comment:
This too. Video is purposefully disabled by default to avoid failures on devices with no video.